run-tests.sh

```diff
@@ -51,7 +51,7 @@ if [[ ${keep_services} -eq 0 ]]; then
 	trap cleanup EXIT
 fi
 
-python -m check_manifest --ignore ".*-requirements.txt"
+python -m check_manifest --ignore ".*-requirements.txt,pyproject.toml"
```

Comment:
😅

Reply:
I can't win: every time I do a `git add -u` it sneaks in :sob: